README.md

```diff
@@ -7,7 +7,7 @@
 
 ![Screenshot](docs/angular-cli-ghpages-deploy.gif)
 
-**Table of contents:**  
```

Comment:
You should install https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig next time! ;-)